packages/components/src/components/Combobox/ComboboxPopover/ComboboxPopover.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { forwardRef } from "react";
+import {
+  ComboboxPopover as AriakitComboboxPopover,
+  ComboboxPopoverProps as AriakitComboboxPopoverProps,
+} from "@ariakit/react";
+import { Box } from "../../../primitives/Box";
+
+export const ComboboxPopover = forwardRef<
+  HTMLDivElement,
+  AriakitComboboxPopoverProps & {
+    children: React.ReactNode;
+  }
+>(({ children, ...props }, ref) => {
+  return (
+    <Box.div
+      as={AriakitComboboxPopover}
+      backgroundColor="bgPrimary"
+      borderRadius="borderRadius20"
+      boxShadow="shadowStrong"
+      display="flex"
+      flexDirection="column"
+      gutter={3}
+      maxHeight="250px"
+      outlineColor={{ focus: "colorBorderPrimary" }}
+      outlineStyle={{ focus: "solid" }}
+      outlineWidth={{ focus: "borderWidth10" }}
```

Comment:
I think `focus` is not working as expected here, I cannot see the blue outline. I had to use the devtools and simulate the focus on the specific html element to see it

Reply:
I used the same styles from talent-web, but now I see we don't need them. The focused element should be an option when navigating down from the input, not the popover.